### DISCUSSION TITLE
Manually constructing a Dynamic Link URL

### DISCUSSION THREAD

**Discussion by mxl** [kreait/firebase-php#455] [Ideas]
Does this library support manual Dynamic Link URL construction as described [here](https://firebase.google.com/docs/dynamic-links/create-manually)?

**Comment by jeromegamez** [comment]
It does 😄

https://firebase-php.readthedocs.io/en/5.7.0/dynamic-links.html

**Comment by mxl** [comment]
@jeromegamez I mean constructing dynamic link not creating new one with suffix. As I see from firebase-php source code it calls Firebase API to create new dynamic link with either unguessable or short suffix but it doesn't have option to construct dynamic link without calling Firebase API as explained [here](https://firebase.google.com/docs/dynamic-links/create-manually).

**Comment by jeromegamez** [comment]
Ah, sorry, I misunderstood 😅. That's currently not possible, but I'll mark this as a feature request, thank you for bringing this up!

**Comment by mxl** [comment]
As a workaround I wrote today a [library](https://github.com/mxl/firebase-dynamic-link-php) which solves my issue. It has an API similar to your library.
